optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -453,9 +453,9 @@ public int hashCode() {
                             it.remove();
                         }
                     }
-                    newResult.entityCount = ConfigUtils.mergeProperty(oldResult.entityCount, newResult.entityCount);
-                    newResult.variableCount = ConfigUtils.mergeProperty(oldResult.variableCount, newResult.variableCount);
-                    newResult.problemScale = ConfigUtils.mergeProperty(oldResult.problemScale, newResult.problemScale);
+                    newResult.entityCount = ConfigUtils.meldProperty(oldResult.entityCount, newResult.entityCount);
```

Comment:
When would a meld (that is not a merge) happen? So when would a.entityCount be null and b.entityCount not be null? I would argue that can only happen when loading older benchmarks, in which the migration code should take care of that (or do nothing and just leave it to become null as merged), and not the agregation code here?